mirobo/containers.py

```diff
@@ -39,6 +39,115 @@ def pretty_time(x: int) -> datetime:
 }
 
 
+class AirPurifierStatus:
+    """Container for status reports from the air purifier."""
+    def __init__(self, data: Dict[str, Any]) -> None:
+        # ['power', 'aqi', 'humidity', 'temp_dec',
+        #  'mode', 'led', 'led_b', 'buzzer', 'child_lock',
+        #  'limit_hum', 'trans_level', 'bright',
+        #  'favorite_level', 'filter1_life', 'act_det',
+        #  'f1_hour_used', 'use_time', 'motor1_speed']
+        self.data = data
+
+    @property
+    def power(self) -> str:
+        return self.data["power"]
```

Comment:
Do you have an example what type of response is expected here? A numerical or a string?